packages/firestore/src/util/async_queue.ts

```diff
@@ -208,12 +218,65 @@ export class AsyncQueue {
     this.enqueue(op);
   }
 
+  /**
+   * Regardless if the queue has initialized shutdown, adds a new operation to the
+   * queue without waiting for it to complete (i.e. we ignore the Promise result).
+   */
+  enqueueAndForgetEvenAfterShutdown<T extends unknown>(
+    op: () => Promise<T>
+  ): void {
+    this.verifyNotFailed();
+    // tslint:disable-next-line:no-floating-promises
+    this.enqueueInternal(op);
+  }
+
+  /**
+   * Regardless if the queue has initialized shutdown, adds a new operation to the
+   * queue.
+   */
+  private enqueueEvenAfterShutdown<T extends unknown>(
+    op: () => Promise<T>
+  ): Promise<T> {
+    this.verifyNotFailed();
+    // tslint:disable-next-line:no-floating-promises
```

Comment:
this shouldn't be necessary.

Reply:
Actually `yarn lint` complains about this.